types/storybook__react/storybook__react-tests.tsx

```diff
@@ -1,17 +1,31 @@
 import * as React from 'react';
-import { storiesOf, setAddon, addDecorator, addParameters, configure, getStorybook, RenderFunction, Story, forceReRender, DecoratorParameters, clearDecorators } from '@storybook/react';
+import {
+    storiesOf,
+    setAddon,
+    addDecorator,
+    addParameters,
+    configure,
+    getStorybook,
+    RenderFunction,
+    Story,
+    forceReRender,
+    load,
+    DecoratorParameters,
+    clearDecorators,
+} from '@storybook/react';
 
 const Decorator = (story: RenderFunction) => <div>{story()}</div>;
 const parameters: DecoratorParameters = { parameter: 'foo' };
 
 forceReRender();
+load(require.context('../src', true, /\.stories\.tsx$/), module);
 
 storiesOf('Welcome', module)
```

Comment:
everything below here is unrelated, right? Best to remove the noise if it's not needed for the PR to land?

Reply:
Yeah a lot of this is just due to some extension in VS Code doing formatting. I'll pull all those formatting changes out.

Reply:
This is also coming from that same pre-commit hook mentioned above.